packages/frontend/src/pages/ModelInspect.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+import { onMount } from 'svelte';
+import { modelsMetadata } from '/@/stores/modelsMetadata';
+import {
+  Table,
+  TableRow,
+  TableColumn,
+  TableSimpleColumn,
+  ErrorMessage,
+  LinearProgress,
+} from '@podman-desktop/ui-svelte';
+import { studioClient } from '/@/utils/client';
+
+export let modelId: string;
+
+type Item = {
+  name: string;
+  value: string;
+};
+
+let items: Item[] = [];
+$: items;
+
+let loading: boolean = true;
+$: loading;
+
+let error: string | undefined = undefined;
+$: error;
+
+const columns = [
+  new TableColumn<Item, string>('property', {
+    renderMapping: item => String(item.name),
+    renderer: TableSimpleColumn,
+  }),
+  new TableColumn<Item, string>('value', {
+    renderMapping: item => String(item.value),
+    renderer: TableSimpleColumn,
+  }),
+];
+const row = new TableRow<Item>({});
+
+onMount(() => {
+  return modelsMetadata.subscribe(metadatas => {
```

Comment:
Why subscribing to a store if the only place where it is written it's here? Would it be simpler to have something like
```
studioClient
        .getModelMetadata(modelId)
        .then(result => {
        items = .....
```

without creating/subscribing to a store which is never updated elsewhere.

Reply:
If you leave the page and come back, this is made for caching it

Reply:
ah ok, please add a comment as a reminder